Formula/ceph-client.rb

```diff
@@ -0,0 +1,103 @@
+class CephClient < Formula
+  desc "Rados and RBD CLIs and libraries of the Ceph project"
+  homepage "https://ceph.com"
+  url "git@github.com:ceph/ceph.git", :using => :git, :tag => "v13.2.1", :revision => "5533ecdc0fda920179d7ad84e0aa65a127b20d77"
+  version "mimic-13.2.1"
+
+  depends_on python if MacOS.version <= :snow_leopard
+
+  depends_on :osxfuse
+  depends_on "openssl" => :build
+  depends_on "ccache" => :build
+  depends_on "cmake" => :build
+  depends_on "cython" => :build
+  depends_on "leveldb" => :build
+  depends_on "nss"
+  depends_on "pkg-config" => :build
+  depends_on "yasm"
+
+  def install
+    ENV.prepend_path "PKG_CONFIG_PATH", "#{Formula["nss"].opt_lib}/pkgconfig"
+    ENV.prepend_path "PKG_CONFIG_PATH", "#{Formula["openssl"].opt_lib}/pkgconfig"
+    system "./do_cmake.sh",
+              "-DCMAKE_CXX_COMPILER=clang++",
+              "-DCMAKE_C_COMPILER=clang",
+              "-DWITH_EMBEDDED=OFF",
+              "-DWITH_MANPAGE=ON",
+              "-DWITH_LIBCEPHFS=OFF",
+              "-DWITH_XFS=OFF",
+              "-DWITH_KRBD=OFF",
+              "-DWITH_LTTNG=OFF",
+              "-DCMAKE_BUILD_TYPE=Debug",
+              "-DWITH_CCACHE=ON",
+              "-DWITH_RADOSGW=OFF",
+              "-DWITH_CEPHFS=OFF",
```

Comment:
WITH_CEPHFS should be `ON` .  The cephfs client libraries are needed for commands like "ceph tell mds.0 ..." to work when using the CLI.